docs/modeling/parallel-fitting.rst

```diff
@@ -171,6 +171,8 @@ to fit all spectra in the cube:
    :include-source:
    :nofigs:
 
+.. doctest-requires:: dask
```

Comment:
If the CI discussion drags on, this one should be uncontroversial and can be a separate PR.

Reply:
https://github.com/astropy/astropy/pull/17060 👀